drizzlepac/haputils/mvm_quality_graphics.py

```diff
@@ -39,18 +47,18 @@
 
 # ----------------------------------------------------------------------------------------------------------------------
 # Module level variables
-color26 = ['#1f77b4', '#393b79', '#aec7e8', '#ff7f0e', '#ffbb78', '#5254a3', '#2ca02c',
-           '#98df8a', '#6b6ecf', '#d62728', '#ff9896', '#9467bd', '#c5b0d5', '#7b4173',
-           '#8c564b', '#c49c94', '#e377c2', '#f7b6d2', '#a55194', '#7f7f7f', '#c7c7c7',
-           '#bcbd22', '#dbdb8d', '#ce6dbd', '#17becf', '#9edae5']
-
-wcs_list = ['FIT_SVM_GAIAEDR3', 'FIT_EVM_GAIAEDR3', 'FIT_REL_GAIAEDR3', 'FIT_IMG_GAIAEDR3',
-            'FIT_SVM_GAIADR2', 'FIT_EVM_GAIADR2', 'FIT_REL_GAIADR2', 'FIT_IMG_GAIADR2',
-            'FIT_SVM_GAIADR1', 'FIT_EVM_GAIADR1', 'FIT_REL_GAIADR1', 'FIT_IMG_GAIADR1',
-            'FIT_SVM_GSC242', 'FIT_EVM_GSC242', 'FIT_REL_GSC242', 'FIT_IMG_GSC242',
-            'FIT_SVM_2MASS', 'FIT_EVM_2MASS', 'FIT_REL_2MASS', 'FIT_IMG_2MASS',
-            'FIT_SVM_NONE', 'FIT_EVM_NONE', 'FIT_REL_NONE', 'FIT_IMG_NONE',
-            'HSC30', 'GSC240']
+color20 = ['#1f77b4', '#393b79', '#aec7e8', '#ffbb78', '#2ca02c',
+           '#6b6ecf', '#d62728', '#ff9896', '#9467bd', '#c5b0d5',
+           '#8c564b', '#e377c2', '#f7b6d2', '#a55194', '#7f7f7f',
+           '#c7c7c7', '#bcbd22', '#dbdb8d', '#ce6dbd', '#9edae5']
+
+wcs_list = {'FIT_SVM_GAIAEDR3': 1, 'FIT_REL_GAIAEDR3': 2, 'FIT_IMG_GAIAEDR3': 4,
```

Comment:
Could you simply import the `wcs_pref_list` from mvm_quality_analysis to prevent duplication of this same information and avoid any potential discrepancies between the two definitions?

Reply:
Addressed by update in [9c5c85c] (https://github.com/spacetelescope/drizzlepac/pull/1421/commits/9c5c85c7a872ef9926af2cd985629ae44082fa4b).